pkg/redis.ts

```diff
@@ -208,6 +209,7 @@ export class Redis {
     this.opts = opts;
     this.enableTelemetry = opts?.enableTelemetry ?? true;
     this.enableAutoPipelining = opts?.enableAutoPipelining ?? false;
+    this.readYourWrites = opts?.readYourWrites ?? true;
```

Comment:
Maybe we should make this opt-in?

Reply:
I've talked with the core team, and they said there's literally no downside other than a small case. Most users wouldn't need to opt-out, and if they do it's possible.